CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
+## [2.2.x] - ???
+### :crab: Changed
+- Dynamically set the default host list provider based on the dialect.
```

Comment:
Not sure if i got it entirely, but what about:

"Dynamically sets the default hostlist provider based on the dialect used. AuroraHostList plugin is not required anymore when using TopologyAwareDatabaseCluster dialect."